website/.prettierrc

```diff
@@ -20,12 +20,11 @@
             }
         },
         {
-            "files": "*.md",
+            "files": ["package.json", "package-lock.json"],
             "options": {
                 "tabWidth": 2,
                 "printWidth": 80,
-                "proseWrap": "always",
-                "htmlWhitespaceSensitivity": "strict"
+                "proseWrap": "always"
```

Comment:
Is this a change that is absolutely necessary?

Reply:
Since this is now the rule applied to the JSON files (see my previous comment on slightly misleading diff), `htmlWhitespaceSensitivity` is just a setting not needed anymore since it's the formatting rules for JSON, not Markdown.